src/content/pt/2022/interoperability.md

```diff
@@ -0,0 +1,395 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: Interoperabilidade
+description: Capítulo de Interoperabilidade do Web Almanac de 2022 cobrindo Compat 2021 (Grid e Flexbox) e Interop 2022 (Formulários, Rolagem, Tipografia e codificações, Diálogo, Contenção, Subgrid, Espaços de cor, Unidades de viewport e Camadas de cascata)
+authors: [bkardell]
+reviewers: [meyerweb, foolip]
+analysts: [rviscomi, kevinfarrugia]
+editors: [tunetheweb]
+translators: [HakaCode]
+bkardell_bio: Brian Kardell é um defensor de desenvolvedores e Representante do Comitê Consultivo W3C na <a hreflang="en" href="https://igalia.com">Igalia</a>, uma contribuidora de padrões, <a hreflang="en" href="https://bkardell.com">blogger</a>. . Ele foi um dos fundadores do Grupo Comunitário da Web Extensível e co-autor do <a hreflang="en" href="https://extensiblewebmanifesto.org">Manifesto da Web Extensível</a>.
+results: https://docs.google.com/spreadsheets/d/1w3GzzTNeKxafFODmjDs6OC2dseNEDDKwUV8KeSgRI1Y/
+featured_quote: A interoperabilidade é um objetivo fundamental dos padrões, mas às vezes não conseguimos alcançá-lo totalmente. Este capítulo começará a fornecer uma atualização anual aos desenvolvedores sobre os esforços para se unirem e melhorar as coisas. Ele abordará o que é novo ou foi aprimorado em termos de interoperabilidade este ano e fornecerá um meio para os implementadores medirem os impactos ao longo do tempo.
+featured_stat_1: 309%
+featured_stat_label_1: The increase in sites using CSS `aspect-ratio` between April 2021 and September 2022
+featured_stat_2: 0.3%
+featured_stat_label_2: The percentage of sites using the newly interoperable `dialog` element as of September 2022
+featured_stat_3: 4%
+featured_stat_label_3: Mobile pages using recently interoperable CSS `containment`. This support is critical for Container Queries.
```

Comment:
Can we translate these?

Reply:
Yes, I was in doubt whether or not I could translate

Reply:
Yes, I was in doubt whether or not I could translate